bindings/python/multibody/joint/joint-derived.hpp

```diff
@@ -33,6 +33,8 @@ namespace pinocchio
         .add_property("idx_v",&get_idx_v)
         .add_property("nq",&get_nq)
         .add_property("nv",&get_nv)
+        .add_property("configuration_limit", &JointModelDerived::hasConfigurationLimit,
```

Comment:
Maybe ``"has_configuration_limit"`` so that the API names match (if not their case)?